### PR TITLE
upgrademanager: skip flaky TestConcurrentMigrationAttempts test

### PR DESCRIPTION
**Pull request by rharding6373**
Epic: None
Informs: #101021

Release note: None

### CODE REVIEW

File: pkg/BUILD.bazel [hunk 1]
```diff
@@ -1057,6 +1057,8 @@ GO_TARGETS = [
     "//pkg/cmd/roachtest/grafana:grafana",
     "//pkg/cmd/roachtest/option:option",
     "//pkg/cmd/roachtest/option:option_test",
+    "//pkg/cmd/roachtest/prometheus/gomock_reflect_1881716363:gomock_reflect_1881716363",
```
Comment (rafiss) [inline pkg/BUILD.bazel:1060]: are these unrelated changes?

**Review by rafiss — APPROVED** [review]

File: pkg/BUILD.bazel (hunk 1 above)
Comment (rharding6373) [reply pkg/BUILD.bazel]: Thanks for catching this. I'm not sure why `dev generate bazel` seems to always put these in.

Comment (rharding6373) [reply pkg/BUILD.bazel]: I reverted those changes.

**Comment by rharding6373** [comment]
TFTR!

bors r+
